xDrip/Scenes/Settings/SettingsUnits/SettingsUnitsPresenter.swift

```diff
@@ -22,7 +23,34 @@ final class SettingsUnitsPresenter: SettingsUnitsPresentationLogic {
     // MARK: Do something
     
     func presentLoad(response: SettingsUnits.Load.Response) {
-        let viewModel = SettingsUnits.Load.ViewModel()
+        let tableViewModel = BaseSettings.ViewModel(sections: [
+            createUnitsSection(response: response)
+        ])
+        
+        let viewModel = SettingsUnits.Load.ViewModel(tableViewModel: tableViewModel)
         viewController?.displayLoad(viewModel: viewModel)
     }
+    
+    func presentSelected(response: SettingsUnits.Select.Response) {
+        let index = GlucoseUnit.allCases.firstIndex(of: response.currentSelectedUnit) ?? 0
+        
+        let viewModel = SettingsUnits.Select.ViewModel(currentSelectedUnitIndex: index)
+        viewController?.displaySelect(viewModel: viewModel)
+    }
+    
+    private func createUnitsSection(response: SettingsUnits.Load.Response) -> BaseSettings.Section {
+        let titles = GlucoseUnit.allCases.map { $0.title }
+        let index = GlucoseUnit.allCases.firstIndex(of: response.currentSelectedUnit) ?? 0
+        
+        return BaseSettings.Section.singleSelection(cells: titles, selectedIndex: index, header: " ", footer: nil, selectionHandler: response.selectionHandler)
```

Comment:
Do we really need an empty header here? If we need an offset, maybe it's worth to globally inset BaseSettings's `tableView` with `contentInset`?